ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/CaseMapper.java

```diff
@@ -113,6 +118,12 @@ public Claim from(CCDCase ccdCase) {
             settlement = settlementMapper.from(ccdCase.getSettlement());
         }
 
+        List<String> features = null;
+        if (ccdCase.getFeatures() != null) {
```

Comment:
can have additional check if the ccdCase length is more than 0

Reply:
ccdCaseFeatures is comma separated string

Reply:
Just question, if it comes with just single comma by mistake what happens?